lib/mm.js

```diff
@@ -260,6 +258,7 @@ function mockRequest() {
  * @param {Number} [delay], response delay time, default is 10.
  */
 exports.http.request = function (url, data, headers, delay) {
+  getAgent(http).__sourceRequest = getAgent(http).request;
   return _request.call(this, http, url, data, headers, delay);
```

Comment:
mmh... indeed. Although it's meaningless if user calls the mock request twice(because the later mock will override the previous one), unable to restore to the original one is unacceptable. Will update the PR.